adp-aso-helm-library/templates/_flexible-servers-db.yaml

```diff
@@ -0,0 +1,26 @@
+{{- define "adp-aso-helm-library.flexible-servers-db.tpl" -}}
+{{- $requiredMsg := include "adp-aso-helm-library.default-check-required-msg" . -}}
+apiVersion: dbforpostgresql.azure.com/v1api20220120preview
```

Comment:
Can you change the version to 'v1api20210601'
https://azure.github.io/azure-service-operator/reference/dbforpostgresql/

Reply:
done